shopinvader/models/shopinvader_product.py

```diff
@@ -99,7 +99,7 @@ def _compute_shopinvader_category(self):
                 domain = record._get_parent_category_domain(product_category)
                 parents = self.env["shopinvader.category"].search(domain)
                 categories |= parents
```

Comment:
seems legit :)

As categories can be a lot, can we drop intersection of recordsets and use a set of ids to improve perf?

Reply:
Do you have perfs issues on this path ? 
May be avoid doing a search in loop will lead to better results?

Reply:
Intersecting browse records is always slower than browsing the same records at once w/ their ids.
Also, here you don't need the recordset at all.

We could use `read_group`. I don't want to block this fix so for now just add a TODO. 
The change for the recordset is easy to do instead: Just do:
```py
category_ids = []
[...]
category_ids += list(self...search([]).ids)
[...]
record.shopinvader_categ_ids = category_ids
￼```
more or less